crates/turborepo-lib/src/commands/login.rs

```diff
@@ -346,14 +365,23 @@ mod test {
             version: "",
         };
 
-        login::login(&mut base).await.unwrap();
+        // Attempt first login to get new token.
+        let first_token = {
+            login::login(&mut base).await.unwrap();
+            base.user_config().unwrap().token().unwrap().to_string()
+        };
+
+        let second_token = {
+            login::login(&mut base).await.unwrap();
+            base.user_config().unwrap().token().unwrap().to_string()
+        };
 
         handle.abort();
 
-        assert_eq!(
-            base.user_config().unwrap().token().unwrap(),
-            turborepo_vercel_api_mock::EXPECTED_TOKEN
-        );
+        // Save token we just got. Run the login again to make sure the token we got
+        // doesn't change.
+        assert_eq!(first_token, turborepo_vercel_api_mock::EXPECTED_TOKEN);
+        assert_eq!(first_token, second_token);
```

Comment:
Is it possible to assert that we didn't hit the the `/turborepo/token` endpoint (or startup the one_shot_server)? The equality check is somewhat dependent on the server's behavior (which can mostly trust, but not _really_ the thing we need to validate here

Reply:
I admit defeat for now. We can revisit how we're doing this at a later time.